ironfish-cli/src/commands/wallet/multisig/participant/create.ts

```diff
@@ -7,7 +7,7 @@ import { IronfishCommand } from '../../../../command'
 import { RemoteFlags } from '../../../../flags'
 
 export class MultisigIdentityCreate extends IronfishCommand {
-  static description = `Create a multisig identity`
+  static description = `Create a multisig participant identity`
```

Comment:
There are a couple of other places in the help text that we refer to `identity`. Should we change those to `participant identity`, too?

Reply:
Yep, updated